Extensions/callback-data.md

```diff
@@ -56,12 +110,12 @@ Content-Type: application/json
         "values": [
             {
                 "recipient": { "value": "Bob Boston" },
-                "template": { "value": "Standard development agreement" },
+                "template": { "value": "Standard development agreement", "href": "https://example.com/standard-development-agreement" },
```

Comment:
The direct "href" prop might be awkward if you in the future want to be able to render anything else than text or anchor.

Not sure what that might be, a button, chips, or something, but perhaps it should be an object with a $type property instead of the raw href.

`"renderAs": {"$type": link, "href":"https://example.com/premium-maintenance-sla"}`

Reply:
good idea!